modules/ROOT/pages/create-outbound-message-flow.adoc

```diff
@@ -1,126 +1,139 @@
-= Create and Configure an Outbound Message Flow
+= Create Outbound Message Flows
 
-You can create and configure an outbound message flow to receive application messages from your backend systems, validate and transform these messages, and then send the translated EDI message to your trading partners either directly, or through 3rd party connections.
+Create outbound message flows to receive application messages from your backend systems, validate and transform these messages to the format your partners expect, and then send them to your partners either directly, or through 3rd party connections.
 
-To send messages to a trading partner through a 3rd party connection, select an endpoint owned by that connection.
+Before you create your first message flow, configure your xref:setup-payload-storage-API.adoc[payload storage API].
 
-== Create an Outbound Message Flow
+To create an outbound message flow:
 
-. From your Anypoint Partner Manager sandbox or production environment, select *Message Flows* in the navigation menu.
+. From the Anypoint Partner Manager navigation menu, select *Message Flows*.
 . Click *New message flow > Send to partner*.
-. Select an existing partner from the list and click *Select* or select *New* to xref:configure-partner.adoc#create-partner[create a trading partner]. +
-A new message flow is created. +
-
-== Configure the Source at Host Endpoint
-
-. Expand the *Source at <host-name>* section.
-. In *Endpoint: Source at <host-name>*, click *Select*.
-. In the *Select an endpoint: Source at <host-name>* window:
-.. To use an existing receiving endpoint:
-... Select the endpoint from the list and click *Select*.  +
-The endpoint is added to the message flow, and a message appears indicating that the endpoint is missing an identifier.
-... Click *Add* to add a unique identifier for the endpoint.
-.. To create a new receiving endpoint:
-... Click *New* next to the search box.
-... From the *Owner* drop-down list, select the endpoint's owner.
-... From the *Protocol* drop-down, select the protocol and configure it according to the protocol you select:
-
-. Click *Save*. +
-The endpoint is added to the message flow.
+. Either select an existing partner or select *New* to create a new partner, as described in xref:create-partner.adoc#[Create a partner].
++
+A new message flow is created.
```

Comment:
Dunno' if we really need this.

Reply:
I still think it's weird to make the configuration part of the creation without specifying that in the title....
Would it work to end the create task here and then follow with a sentence and a new list of links:

```suggestion
After you create a new message flow, customize it by performing the following tasks:
```
number list or steps, as you think most appropriate